src/components/homepage/chat-overview.tsx

```diff
@@ -1,81 +1,112 @@
 "use client";
 
+import { Input } from "~/components/ui/input";
 import { useQuery } from "convex/react";
 import { api } from "../../../convex/_generated/api";
 import { useUser } from "@clerk/nextjs";
+import { useState } from "react";
 import { Avatar, AvatarFallback } from "~/components/ui/avatar";
+import Badge from "~/components/ui/badge";
 import { NotebookText } from "lucide-react";
-import { Check } from "lucide-react";
-import { MousePointerClick } from "lucide-react";
 
-export function Chats() {
+interface User {
+  clerkId: string;
+  username: string;
+  firstName?: string;
+  lastName?: string;
+}
+
+interface Chat {
+  users: User[];
+  // other properties of chat
+}
+
+const Chats: React.FC = () => {
   const chats = useQuery(api.chats.getChats);
   const clerkUser = useUser();
+  const [searchTerm, setSearchTerm] = useState("");
 
-  return (
-    <>
-      <div className="flex justify-center">
-        <div className="mt-20 flex w-full flex-col items-center lg:w-1/3">
-          {chats?.map((chat, index) => {
-            if (chat.support) {
-              return (
-                <>
-                  <div
-                    key={index}
-                    className="flex w-full items-center justify-start gap-3 border-t-2 border-input py-6 pl-11 lg:mr-16 lg:border-0"
-                  >
-                    <Avatar className="text-white">
-                      <AvatarFallback>C</AvatarFallback>
-                    </Avatar>
-                    <p className="text-xl font-bold">Chat.io</p>
-                    <p className="flex rounded-sm bg-blue-400 p-1 pr-2 text-sm font-medium">
-                      {" "}
-                      <Check className="h-5" /> Support
-                    </p>
-                    <div className="absolute ml-60 mt-20">
-                      <MousePointerClick className="mb-1 ml-6 animate-pulse" />
-                      <p>Click here</p>
-                    </div>
-                  </div>
-                </>
-              );
-            }
-            chat.users = chat.users.filter(
-              (user) => user.clerkId.split("|").pop() !== clerkUser.user?.id,
-            );
+  // Filterfunktion für Benutzer basierend auf clerkId
+  // Filterfunktion für Benutzer basierend auf clerkId
+  const predefinedUsers: User[] = [
```

Comment:
Delete this. We already have them in the backend. They get returned from useQuery(api.chats.getChats);

Look at this to see how I've done it: https://github.com/FleetAdmiralJakob/chat-io/blob/main/src%2Fcomponents%2Fhomepage%2Fchat-overview.tsx